tests/qa_automation/qa_run.pm

```diff
@@ -0,0 +1,100 @@
+package qa_run;
```

Comment:
I think moving qa_automation specific stuff to lib is ugly too - I have the same in firefox.pm and sle11_firefox.pm in x11/.

I don't want firefox.pm in lib/ either